lib/tartarus/archive_strategy.rb

```diff
@@ -1,5 +1,5 @@
 class Tartarus::ArchiveStrategy
-  def for(strategy_name)
+  def for(strategy_name, batch_size)
```

Comment:
can we also make it a keyword argument for a batch_size? Maye having 0 as the default would be helpful here as somehow now, everyone has to pass batch_size even for strategies that don't use this argument